src/main/webapp/js/components/interface/drawer/DrawerButton.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Button component used in the tabbed drawer to 
+ * select the child component to display.
+ *
+ *  @author Dario Del Piano
+ */
+
+define(function (require) {
+
+    var React = require('react');
+    require('./TabbedDrawer.less');
+
+
+    class DrawerButton extends React.Component {
+        constructor(props) {
+            super(props);
+            this.state = {
+                mouseOver: false,
+                buttonActived: false}
+
+            this.activeButton = this.activeButton.bind(this);
+        }
+
+        // function called onClick, recall the parent function passed as prop to display the right child
+        // the props labelKey is passed as parameter to the TabbedDrawer component to let it known
+        // which button has been selected.
+        activeButton() {
+            if((this.props.labelKey === this.props.buttonSelected) && this.props.drawerOpened)
+                this.setState({buttonActived: true,
+                               mouseOver: false});
+            else 
+                this.setState({buttonActived: false,
+                               mouseOver: false});
```

Comment:
`this.setState({buttonActived: this.props.labelKey === this.props.buttonSelected, mouseOver: false});`